### PR TITLE
Re-enable macOS builds for AMD64

### PR DESCRIPTION
**Pull request by odkr**

### CODE REVIEW

File: .github/workflows/manual.yml [hunk 1]
```diff
@@ -4,14 +4,15 @@ on:
   create:
     tags:
       - v*
+  workflow_dispatch:
```
Comment (tasn) [inline .github/workflows/manual.yml:7]:
```suggestion
  workflow_dispatch:
```

**Comment by tasn** [comment]
Is this ready to be merged? I made one change (to remove your test code to drive it as a workflow).

**Comment by tasn** [comment]
(the lint was fixed in master btw)

**Comment by tasn** [comment]
Are there Apple Sillicon runners? If so, please just add it as another matrix item (in a new PR)!

**Comment by odkr** [comment]
> Are there Apple Sillicon runners? If so, please just add it as another matrix item (in a new PR)!

macOS 14 appears to be an Apple Silicon runner (emphasis on "appears", I spotted files that contain "arm64" in the output). 

However, etebase doesn't build on that runner (for the reasons I gave in #328). I can get it to build with the etebase version from the etebase repository, but (a) those would be very different build instructions and (b) I’m not sure that build even works; I can't get either Thunderbird or or macOS to connect to etesync-dav when I use it. Shall we move that discussion to a ticket in etebase? Or a separate ticket here?

**Comment by tasn** [comment]
Ah I forgot about that ticket.

Yeah, let's do a ticket for building etebase-py on arm at the appropriate repo. Thanks!

**Comment by odkr** [comment]
> Yeah, let's do a ticket for building etebase-py on arm at the appropriate repo. Thanks!

Here you go: https://github.com/etesync/etebase-py/issues/56

**Pull request merged**